tools/docker-compose-minikube/minikube/templates/api_credential.json.j2

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Minikube",
+    "credential_type": "16",
```

Comment:
This seems likely to break / not be consistent. We'll probably want to query the API to get the ID.

Reply:
Yes, the new approach addresses this ugly part ;)